src/version.rc

```diff
@@ -0,0 +1,38 @@
+
+#include "winresrc.h"
+
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// Version
+//
+
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION 3,0,0,0
+ PRODUCTVERSION 3,0,0,0
+ FILEFLAGSMASK 0x3fL
+ FILEFLAGS 0x0L
+ FILEOS 0x40004L
+ FILETYPE 0x2L
+ FILESUBTYPE 0x0L
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "CompanyName", "\0"
+            VALUE "FileDescription", "SDL_net\0"
+            VALUE "FileVersion", "3, 0, 0, 0\0"
+            VALUE "InternalName", "SDL_net\0"
+            VALUE "LegalCopyright", "Copyright (C) 2023 Sam Lantinga\0"
+            VALUE "OriginalFilename", "SDL3_net.dll\0"
```

Comment:
This is [`SDL_image.dll` for SDL3_image](https://github.com/libsdl-org/SDL_image/blob/7ccf8c81f2afb69a7d2d1210555f8bcacf9ba498/src/version.rc#L29). Is that correct? I changed it into `SDL3_net.dll` here.

Reply:
I don't know what the policy is on these. SDL3_mixer also has this as "SDL_mixer.dll" at the moment, though, fwiw.

Reply:
This data can be accessed through [`VerQueryValueA`](https://learn.microsoft.com/en-us/windows/win32/api/winver/nf-winver-getfileversioninfoa).
[This site says](https://support.smartbear.com/testcomplete/docs/reference/program-objects/aqfileversioninfo/originalfilename.html):
> The OriginalFilename property returns the original name of the file (without the path). This is the name that the file was created with. So, you can use this property to check if the file has been renamed.


So I think the rename is correct. (Not that it matters :p )

Reply:
I guess let's fix this in SDL_image, then.  :)